docs/resources/samples/filter-table-get-visible-range.md

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
After reading through the whole sample, I realize this creates a Map from an Excel table, which seems both complex and worthy of up-front mention.

Reply:
Perhaps the sample could be remade into two parts: one to show this Table -> Map conversion and one to show applying that logic to a filtered table about stations.

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)